src/NClient.Providers/NClient.Providers.Serialization.Protobuf.ProtobufNet/Extensions/UsingProtobufSerializationExtensions.cs

```diff
@@ -0,0 +1,65 @@
+using NClient.Common.Helpers;
+
+namespace NClient.Providers.Serialization.Protobuf.ProtobufNet.Extensions
```

Comment:
Extensions for creating a client are better defined in the namespace in which the client's builder is defined, like this:
```
// ReSharper disable once CheckNamespace
namespace NClient
```
It is needed for IDE to suggest this method together with the builder methods

Reply:
`ProtobufSerializationExtensions` should also be defined in the `NClient` namespace

Reply:
Fixed